engine/table/src/main/java/io/deephaven/engine/table/impl/select/SwitchColumn.java

```diff
@@ -129,7 +129,7 @@ public String toString() {
     }
 
     public SelectColumn getRealColumn() {
-        return realColumn;
+        return Require.neqNull(realColumn, "realColumn");
```

Comment:
There's a bit of inconsistency in this. Some methods check it, others don't, and none are going to give an appropriate error message. Maybe the other accesses should thread through `getRealColumn()`, and maybe it should have a better validation, like:
```
        if (realColumn == null) {
            throw new IllegalStateException(
                    "Real column is not available until this SwitchColumn is initialized; ensure that initInputs or initDef has been called first"
            );
        }
```

Reply:
Good idea.